js/main.js

```diff
@@ -117,13 +122,14 @@ function updateFeedback(response, player) {
     chunk = `
       <span>${response.toUpperCase()}! ${playerName} loses the game!</span>
     `
+    // if bad slap and single deal === ran out of cards and then slapped something other than a jack
+    //   so the player who slapped bad is loser and other player is winner
     player === 'player1' ? game.loser = 'player1' : game.loser = 'player2'
     player === 'player1' ? game.winner = 'player2' : game.winner = 'player1'
 
-    game.updateWinCount(game.winner)
+    // game.updateWinCount(game.winner)
 
     game.adjustMiddlePile(game.winner)
-    game.resetWholeDeck();
     checkLocalStorage();
 
     player1front.classList.remove('hidden');
```

Comment:
line 149 - if the concatenate idea I had above works, you could get rid of that if/else block and do the same thing there.

Reply:
We talked about this and I tried all the ideas Taryn had and couldn't get the concatenation to work for those.